internal/workflow_testsuite.go

```diff
@@ -83,6 +84,15 @@ type (
 		runFn        func(args mock.Arguments)
 		waitDuration func() time.Duration
 	}
+
+	// TestUpdateCallback is a basic implementation of the UpdateCallbacks interface for testing purposes.
+	// Tests are welcome to implement their own version of this interface if they need to test more complex
+	// update logic. This is a simple implementation to make testing basic Workflow Updates easier.
+	TestUpdateCallback struct {
+		accept   func()
```

Comment:
These fields are private so you also need to make these public so the type can be constructed. To make them public you need to capitalize the first letter. I would also suggest prefixing them with `On` to be a bit more consistent with the other test environment API.

Reply:
Thanks :) tested with https://github.com/temporalio/samples-go/pull/371, hopefully not missing anything else